internal/pkg/prom/prom.go

```diff
@@ -373,7 +394,6 @@ func (c *Client) Store(ctx context.Context, req []byte) error {
 	for k, v := range c.headers {
 		httpReq.Header.Set(k, v)
 	}
-	httpReq = httpReq.WithContext(ctx)
```

Comment:
This was a no-op, as the httpReq's context is reset a few lines below.

I think it's correct to ignore the input ctx, as we're using small timeouts anyway, and we don't want to cancel an in-flight push, but highlighting this just in case it was a mistake.